src/runtime_src/core/edge/tools/xbutil/xbutil.h

```diff
@@ -35,7 +35,7 @@
 #include "core/edge/include/zynq_ioctl.h"
```

Comment:
Copyright date (line 2) needs to be updated.  Being that we are AMD, you might wish to add the following line:
`Copyright (C) 2022 Advanced Micro Devices, Inc.`